### PR TITLE
User story 18

### LINKED ISSUE #16: User Story 18, User can delete a book review
```
As a Visitor,
When I visit a user's show page,
I see a link next to each review to delete the review.
When I delete a review I am returned to the user's show page
Then I should no longer see that review.
```

DO THESE STORIES FIRST:
- User Story 13, Add a Review for a Book
- User Story 16, User Show Page

### Checklist:

- [x] base code tests are written
- [x] base code is written to pass tests
- [x] edge case ("sad path") tests written
- [x] edge case code written
- [x] code is reviewed
- [x] branch is merged
- [x] all tests still pass

child of #9

child of #33

depends on #16

depends on #11

### PR DESCRIPTION
**Pull request by DavisC0801**
This PR...
resolves #16 (Delete a review)
Adds a button to each review on a user's show page.
Removes a review from the database

### CODE REVIEW

File: spec/features/reviews/destroy_spec.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  before(:each) do
+    book_1 = Book.create!(title: "The Frozen Deep", page_count: 106, year_published: 1874, thumbnail: "https://images.gr-assets.com/books/1328728986l/1009218.jpg")
+    book_2 = Book.create!(title: "To Kill a Mockingbird", page_count: 281, year_published: 1960, thumbnail: "https://upload.wikimedia.org/wikipedia/en/7/79/To_Kill_a_Mockingbird.JPG")
+    @user_1 = User.create(username: "Chris Davis")
+    @review_1 = book_1.reviews.create!(title: "This book rocks!", rating: 5, text: "Read it!", user: @user_1)
+    @review_2 = book_2.reviews.create!(title: "This book sucks!", rating: 1, text: "Don't read it!", user: @user_1)
+    @review_3 = book_1.reviews.create!(title: "It's OK.", rating: 3, text: "Meh", user: @user_1)
+  end
+  describe "when I visit a user's show page" do
+    it "shows me a link to delete each review" do
```
Comment (chakeresa) [inline spec/features/reviews/destroy_spec.rb]:
```suggestion
    it "shows me a button to delete each review" do
```

**Comment by chakeresa** [comment]
Looks good! The only suggestion I had was to change "link" to "button" in your test's description, so I went ahead and did it. Merging...
